lerna.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
This tells Lerna that we are using yarn to manage the project, and that we are using yarn workspaces.  Lerna has first class support for workspaces.